services/auth-service/src/auth/user-repository.ts

```diff
@@ -29,7 +29,13 @@ export class UserRepository {
 
   public create(data: CreateUserArgs) {
     return this.prismaService.user.create({
-      data,
+      data: {
+        id: data.id,
```

Comment:
@nurcinozer we can just pass `data` from args. we want to control fields through `CreateUserArgs` type, so that we dont have to type every field in here also.